dms/migrations/12.0.5.0.0/end-migration.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    directories = (
+        env["dms.directory"]
+        .sudo()
+        .search(
+            [("storage_id.save_type", "=", "attachment"), ("group_ids", "!=", False)]
+        )
+    )
+    directories.write({"group_ids": False})
```

Comment:
Why are you removing the content of this field for existing directories? Shouldn't they be only if the check is marked? And I think this should be done with SQL as well.

Reply:
Finally not exists any check in storage because attachment storage access is according to access from record related or model related. Now, it's necessary to not exist groups in directories related and is not possible to assign groups in these directories from UI.

I have set these migration process to reset all fields related (`complete_groups_ids` and groups according to all files from these directories).

Reply:
Talked directly and clarified.